oss-internship-2020/pffft/.gitignore

```diff
@@ -0,0 +1,6 @@
+.hg/
```

Comment:
Do we know where this .hg is coming from? Does it come from .vscode, or is it just some old unnecessary directory. Maybe it'd be safe to remove this dir (seems to be coming from the mercurial source control tools) in your local directory, and it'll never be recreated?

In other words, not sure if this line is needed here.